src/interfaces/fees.ts

```diff
@@ -1,31 +1,24 @@
-import BigNumber from "bignumber.js";
-
 import { ChainId } from "../chain";
 import { ServiceInterface } from "../common";
 import { PROTOCOL_FEES } from "../services/subgraph/queries";
 import { FeesResponse } from "../services/subgraph/responses";
 import { Usdc } from "../types";
+import { toBN } from "../utils";
 
-const BigZero = new BigNumber(0);
+const BigZero = toBN(0);
```

Comment:
Still wondering why we have this? Could we just do it in line?

```typescript
return BigZero.toFixed(0);

// vs

return toBN(0).toFixed(0);
```

Reply:
I think so, why not?  Unsure what's special about `new BigNumber(0)` that deserves a `const` :)

Reply:
Analogous to use 0, I wouldn't do
```typescript
const Zero = 0;
```
then use it.

Plus, if we go with `const` we should stick to the conventions and do `const BIG_ZERO = toBN(0);` but I really don't see the need for it. What are your thoughts?